sources/packages/forms/src/form-definitions/studentexceptions.json

```diff
@@ -411,11 +414,40 @@
             }
           ],
           "tableView": false,
-          "type": "well",
           "hideLabel": true,
-          "lockKey": true,
+          "lockKey": true
+        },
+        {
+          "label": "Current Year Parent Income Exception",
+          "key": "currentYearParentIncomeApplicationException",
```

Comment:
In runtime the component key of the component is created with `1` appended in the end. I believe it is because it conflicts with the key of HTML component in it. It is important that the well component has the key matching with the hidden component in application. Can we change the key of html component?

<img width="619" height="436" alt="image" src="https://github.com/user-attachments/assets/aea52032-5b3f-48fd-bb74-9463de3e4492" />

Reply:
Changed the html key, thanks!